docs/src/guides/ibexa/deploy.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Get started
+weight: -10
+description: Learn how to use Ibexa DXP on Platform.sh.
+---
+
+Ibexa DXP is a Composer-based PHP CMS, and as such fits well with the Platform.sh model.
+As a Symfony-based application its setup is very similar to Symfony.
+
+Ibexa DXP has come pre-configured for use with Platform.sh since its predecessor, eZ Platform 1.13.
+Version 2.5 and later is recommended.
+Those are the only versions that are supported.
+Appropriate Platform.sh configuration files are included in the Ibexa DXP application itself, but may be modified to suit your particular site, if needed.
+
+## Cache and sessions
```

Comment:
Redis should have its own sub-page to be consistent with the global structure.

Reply:
I don't think that structure is something that's necessary. It makes sense in certain cases, but I'm not sure we have to implement it globally without investigating whether it's the right structure.

Reply:
Fair point :)